src/psyclone/nemo.py

```diff
@@ -705,13 +607,47 @@ def match(node):
             return False
         # We are expecting something like:
         #    array(:,:,jk) = some_expression
-        # so we check the left-hand side...
+        # but we have to beware of cases like the following:
```

Comment:
There is a typo in the docstring - it says represet rather than represent

Reply:
Well spotted.